pkg/api/unversioned/well_known_labels.go

```diff
@@ -16,5 +16,5 @@ limitations under the License.
 
 package unversioned
 
-const LabelZoneFailureDomain = "kubernetes.io/zone-failure-domain"
-const LabelZoneRegion = "kubernetes.io/zone-region"
+const LabelZoneFailureDomain = "failure-domain.alpha.kubernetes.io/zone"
```

Comment:
@quinton-hoole out of curiosity, does the "alpha" mean the failure-domain is an alpha feature?